build-system/server/app-index/test/test-amphtml-helpers.js

```diff
@@ -38,6 +38,7 @@ describe('devdash', () => {
     describe('AmpDoc', () => {
 
       it('fails without args', () => {
+        // @ts-ignore
```

Comment:
Since these are JS files it feels weird to have ts-ignore comments in here. I know you added it your previous PR but I'm having second thoughts on the usefulness.. what do you think?

Reply:
I've added this file and others which included ts-ignore to the tsconfig exludes.